benchmarks/plot_runs.py

```
@@ -0,0 +1,57 @@
+        overall_mean + overall_std,
+        f"Overall Mean: {overall_mean:.2f}\nOverall Std: {overall_std:.2f}",
+        color="red",
+        fontsize=10,
+        verticalalignment="bottom",
+    )
+
+    plt.xlabel("Index / Time Step")
+    plt.ylabel("Value")
+    plt.title(f"Row-wise Mean with Standard Deviation for {benchmark}")
+    plt.legend()
+    plt.grid(True)
```

Ruff (0.11.9)

Boolean positional value in function call

(FBT003)